### DISCUSSION TITLE
scriptlet: cleanup lyrics from bad import

### DISCUSSION THREAD

**Discussion by toppk** [beetbox/beets#4364] [Show and tell]
Notice your lyrics have a some non-lyrics text, like ("Tekst piosenki" and "Dodaj interpretację").  in the lyrics plugin, there is webscraping, which caused some additional text to be pulled in.  This pull request fixes:
   https://github.com/beetbox/beets/pull/4229

But I didn't see any simple way to clean up the tags.  I wrote this simple script to check every tracks and remove the text if it is there.

This is working on python3.10 on linux where the lyrics are stored both inside the audiofile and in the database.  Please ensure that is what you want before running this script.  **as written, it will place the lyrics inside the actual mp3 file.**

https://gist.github.com/toppk/0384177a93cf9c9739ead5e280f3dea1

**Comment by sampsyo** [comment]
Looks super cool. Thank you for contributing this!!
